templates/profile.html

```diff
@@ -1,40 +1,44 @@
 {% extends 'base.html' %}
 
 {% block body %}
+{% block body_class %}profile{% endblock %}
```

Comment:
This should be outside of the `{% block body %}`. Probably above.

Reply:
ok